### PR TITLE
Fix some flaws of Russian translation

### LINKED ISSUE #1067: Proofread Russian version of README
There are some flaws in Russian translation of README (mostly grammar and punctuation errors), we need to correct it.

### PR DESCRIPTION
**Pull request by DuckerMan**
Resolves #1067

### CODE REVIEW

**Comment by sharkdp** [comment]
Thank you very much. I am happy to merge this, but I can not review it. @gsomix? Otherwise, I'm just going to merge it in a few das.

**Comment by gsomix** [comment]
@DuckerMan @sharkdp Yeah, I'll review it.

File: doc/README-ru.md [hunk 1]
```diff
@@ -28,20 +28,20 @@
 
 ![Пример интеграции с Git](https://i.imgur.com/azUAzdx.png)
 
-### Показать символы, которые не возможно отобразить
+### Показать символы, которые невозможно отобразить
```
Comment (gsomix) [inline doc/README-ru.md:31]:
```suggestion
### Показать непечатаемые символы
```

File: doc/README-ru.md [hunk 2]
```diff
@@ -28,20 +28,20 @@
...
 
 Вы можете использовать `-A` / `--show-all` флаг чтобы показать символы, которые невозможно напечатать:
 
 ![Строка с неотображемыми символами](https://i.imgur.com/X0orYY9.png)
 
 ### Автоматическое разделение текста
 
-`bat` умеет траслировать вывод в `less` если вывод слишком большой, чтобы уместится на экране целиком
+`bat` умеет траслировать вывод в `less` если он слишком большой, чтобы уместится на экране полностью
```
Comment (gsomix) [inline doc/README-ru.md:39]:
```suggestion
`bat` умеет перенаправлять вывод в `less`, если вывод не помещается на экране полностью.
```

File: doc/README-ru.md [hunk 3]
```diff
@@ -28,20 +28,20 @@
...
 
 ### Объединение файлов
 
-О..вы так же можете объединять файлы :wink:. Всегда когда
-`bat` обнаруживает неинтерактивный терминал (например когда вы направляете вывод в файл или процесс), `bat` будет работать как вставная замена `cat` и откатится к выводу это файла как обычного текста
+О..Вы так же можете объединять файлы :wink:. Всегда когда
```
Comment (gsomix) [inline doc/README-ru.md:43]:
```suggestion
О... Вы также можете объединять файлы :wink:. Когда
```

File: doc/README-ru.md [hunk 4]
```diff
@@ -28,20 +28,20 @@
...
+`bat` обнаруживает неинтерактивный терминал (например когда вы направляете вывод в файл или в процесс), `bat` будет работать как `cat` и откатится к выводу это файла как обычного текста (без поддержки синтаксиса)
```
Comment (gsomix) [inline doc/README-ru.md:44]:
```suggestion
`bat` обнаружит неинтерактивный терминал (например, когда вы перенаправляете вывод в файл или процесс), он будет работать как утилита `cat` и выведет содержимое файлов как обычный текст (без подсветки синтаксиса).
```

File: doc/README-ru.md [hunk 5]
```diff
@@ -63,7 +63,7 @@
 > curl -s https://sh.rustup.rs | bat
 ```
 
-Читает из stdin и указываем что это за язык
+Читаем из stdin и указываем что это за язык
```
Comment (gsomix) [inline doc/README-ru.md:66]:
```suggestion
Прочитать из stdin с явным указанием языка
```

File: doc/README-ru.md [hunk 6]
```diff
@@ -110,11 +110,11 @@ batgrep needle src/
 
 #### `tail -f`
 
-`bat` может быть использован с  `tail -f` чтобы просматривать файл вместе с выделением синтаксиса
+`bat` может быть использован вместе с  `tail -f` чтобы просматривать файл вместе с выделением синтаксиса
```
Comment (gsomix) [inline doc/README-ru.md:113]:
```suggestion
`bat` может быть использован вместе с  `tail -f`, чтобы выводить содержимое файла с подсветкой синтаксиса в реальном времени.
```

File: doc/README-ru.md [hunk 7]
```diff
@@ -110,11 +110,11 @@ batgrep needle src/
...
 ```bash
 tail -f /var/log/pacman.log | bat --paging=never -l log
 ```
-Заметье, что мы отключаем пэйджинг чтобы это заработало. Мы так-же указываем синтаксис (`-l log`), так как он не может быть определен в данном случае.
+Заметьте, что мы отключаем пэйджинг чтобы это заработало. Мы так-же указываем синтаксис (`-l log`), так как он не может быть определен в данном случае.
```
Comment (gsomix) [inline doc/README-ru.md:117]:
```suggestion
Заметьте, что мы должны отключить пэйджинг, чтобы это заработало. Мы также явно указали синтаксис (`-l log`), так как он не может быть автоматически определен в данном случае.
```

File: doc/README-ru.md [hunk 8]
```diff
@@ -170,17 +170,17 @@ man 2 select
 apt install bat
 ```
 
-Если вы установили `bat` таким образом, то бинарный файл может быть установлен как `batcat` заместо `bat` (из-за [имя конфлиует с другим пакетом](https://github.com/sharkdp/bat/issues/982)). Вы можете сделать симлинк или алиас `bat -> batcat` чтобы предотвратить проблемы которые могут произойти с другими дистрибутивами.
+Если вы установили `bat` таким образом, то бинарный файл может быть установлен как `batcat` заместо `bat` (из-за [конфликта имени с другим пакетом](https://github.com/sharkdp/bat/issues/982)). Вы можете сделать симлинк или алиас `bat -> batcat` чтобы предотвратить проблемы которые могут произойти с другими дистрибутивами.
```
Comment (gsomix) [inline doc/README-ru.md:173]:
```suggestion
Если вы установили `bat` таким образом, то бинарный файл может быть установлен как `batcat` заместо `bat` (из-за [конфликта имени с другим пакетом](https://github.com/sharkdp/bat/issues/982)). Вы можете сделать симлинк или алиас `bat -> batcat`, чтобы предотвратить проблемы, которые могут произойти с другими дистрибутивами.
```

File: doc/README-ru.md [hunk 9]
```diff
@@ -170,17 +170,17 @@ man 2 select
...
 mkdir -p ~/.local/bin
 ln -s /usr/bin/batcat ~/.local/bin/bat
 ```
 
-### Ubuntu (С помощью самоно нового `.deb` пакета)
+### Ubuntu (С помощью самого нового `.deb` пакета)
 *... и другие дистрибутивы Linux основанные на Debian*
 
-Если пакет еще не был предоставлен в вашем Ubuntu/Debian или вы хотите самую последнюю версия `bat`, вы можете скачать самый последний `deb` пакет отсюда:
+Если пакет еще не был предоставлен в вашем Ubuntu/Debian дистрибутиве или вы хотите самую последнюю версия `bat`, вы можете скачать самый последний `deb` пакет отсюда:
```
Comment (gsomix) [inline doc/README-ru.md:183]:
```suggestion
Если пакет еще недоступен в вашем Ubuntu/Debian дистрибутиве, или вы хотите установить самую последнюю версию `bat`, то вы можете скачать самый последний `deb`-пакет отсюда:
```

File: doc/README-ru.md [hunk 10]
```diff
@@ -340,7 +340,7 @@ ansible-galaxy install aeimer.install_bat
 
 ### Из исходников
 
-Если вы желаете установить `bat` из исходников, вам понадобится Rust 1.40 или выше. Далее, вы должны использовать `cargo` чтобы все скомпилировать:
+Если вы желаете установить `bat` из исходников, вам понадобится Rust 1.40 или выше. После этого, вы должны использовать `cargo` чтобы все скомпилировать:
```
Comment (gsomix) [inline doc/README-ru.md:343]:
```suggestion
Если вы желаете установить `bat` из исходников, вам понадобится Rust 1.40 или выше. После этого вы должны использовать `cargo`, чтобы все скомпилировать:
```

File: doc/README-ru.md [hunk 11]
```diff
@@ -353,13 +353,13 @@ cargo install --locked bat
...
 
-Если вы хотите просто просмотреть темы, используйте следующую команду (вам понадобится [`fzf`](https://github.com/junegunn/fzf) для этого):
+Если вы хотите просто просмотреть темы, используйте следующую команду (для этого вам понадобится [`fzf`](https://github.com/junegunn/fzf)):
 ```bash
 bat --list-themes | fzf --preview="bat --theme={} --color=always /путь/к/файлу"
 ```
 
-`bat` отлично смотрится на темном фоне. Однако, если ваш терминал использует светлую тему, такие темы как `GitHub` или `OneHalfLight` будут смотрется куда лучше!
-Вы так-же можете использовать новую тему, для этого перейдите [в раздел добавления тем](https://github.com/sharkdp/bat#adding-new-themes).
+`bat` отлично смотрится на темном фоне. Однако, если ваш терминал использует светлую тему, такие темы как `GitHub` или `OneHalfLight` будут смотреться куда лучше!
```
Comment (gsomix) [inline doc/README-ru.md:361]:
```suggestion
`bat` отлично смотрится на темном фоне. Однако если ваш терминал использует светлую тему, то такие темы как `GitHub` или `OneHalfLight` будут смотреться куда лучше!
```

File: doc/README-ru.md [hunk 12]
```diff
@@ -353,13 +353,13 @@ cargo install --locked bat
...
+Вы так-же можете использовать новую тему, для этого перейдите [в раздел добавления тем](https://github.com/sharkdp/bat#добавление-новых-тем).
```
Comment (gsomix) [inline doc/README-ru.md:362]:
```suggestion
Вы также можете использовать новую тему, для этого перейдите [в раздел добавления тем](https://github.com/sharkdp/bat#добавление-новых-тем).
```

File: doc/README-ru.md [hunk 13]
```diff
@@ -389,9 +389,9 @@ git clone https://github.com/tellnobody1/sublime-purescript-syntax
 bat cache --build
 ```
 
-Наконец, используйте `bat --list-languages` чтобы проверить, доступны ли новые языки.
+Теперь, вы можете использовать `bat --list-languages` чтобы проверить, доступны ли новые языки.
```
Comment (gsomix) [inline doc/README-ru.md:392]:
```suggestion
Теперь вы можете использовать `bat --list-languages`, чтобы проверить, доступны ли новые языки.
```

File: doc/README-ru.md [hunk 14]
```diff
@@ -389,9 +389,9 @@ git clone https://github.com/tellnobody1/sublime-purescript-syntax
...
 
-Если когда-нибудь вы заходите вернутся к настройкам по умолчанию, введите
+Если когда-нибудь вы заходите вернуться к настройкам по-умолчанию, введите
```
Comment (gsomix) [inline doc/README-ru.md:394]:
```suggestion
Если когда-нибудь вы заходите вернуться к настройкам по умолчанию, введите
```

File: doc/README-ru.md [hunk 15]
```diff
@@ -428,14 +428,14 @@ export BAT_PAGER="less -RF"
 
 Так-же вы можете использовать [файл конфигурации](https://github.com/sharkdp/bat#configuration-file) `bat` для изменения. (флаг `--pager`).
 
-**Внимание**: По умолчанию, `less` является пейджером (без каких либо аргументов),
+**Внимание**: По умолчанию, пейджером является`less` (без каких либо аргументов),
```
Comment (gsomix) [inline doc/README-ru.md:431]:
```suggestion
**Внимание**: По умолчанию пейджером является`less` (без каких-либо аргументов),
```

File: doc/README-ru.md [hunk 16]
```diff
@@ -428,14 +428,14 @@ export BAT_PAGER="less -RF"
...
 `bat` задаст следующие флаги для пейджера:
  `-R`/`--RAW-CONTROL-CHARS`,
 `-F`/`--quit-if-one-screen` и `-X`/`--no-init`. Последний флаг(`-X`) используется только для `less`, чья версия раньше 530.
 
 Флаг `-R` нужен чтобы корректно воспроизвести ANSI цвета. Второй флаг (`-F`) говорит 
 `less` чтобы тот сразу-же завершился, если размер вывода меньше чем вертикальный размер терминала.
-Это удобно для небольших файлов, т.к вам не надо каждый раз нажимать `q` чтобы выйти из пейджера. Третий флаг (`-X`) нужен чтобы исправить баг с `--quit-if-one-screen` в старых версиях `less`. К сожалению, это так-же блокирует возможность использования там колеса мышки.
+Это удобно для небольших файлов, т.к вам не надо каждый раз нажимать `q` чтобы выйти из пейджера. Третий флаг (`-X`) нужен чтобы исправить баг с `--quit-if-one-screen` в старых версиях `less`. К сожалению, это так-же блокирует возможность использования колеса мышки.
```
Comment (gsomix) [inline doc/README-ru.md:438]:
```suggestion
Это удобно для небольших файлов, так как вам не надо каждый раз нажимать `q`, чтобы выйти из пейджера. Третий флаг (`-X`) нужен для того, чтобы исправить баг с `--quit-if-one-screen` в старых версиях `less`. К сожалению, это блокирует возможность использования колеса мышки.
```

File: doc/README-ru.md [hunk 17]
```diff
@@ -467,7 +467,7 @@ bat --generate-config-file
 
 ### Формат
 
-Файл конфигурации - это всего лишь набор аргументов. Введите `bat --help` чтобы просмотреть список всех возможных флагов и аргументов. Так-же, вы можете заккоментировать строчку с помощью `#`.
+Файл конфигурации - это всего лишь набор аргументов. Введите `bat --help` чтобы просмотреть список всех возможных флагов и аргументов. Так-же, вы можете закомментировать строчку с помощью `#`.
```
Comment (gsomix) [inline doc/README-ru.md:470]:
```suggestion
Файл конфигурации - это всего лишь набор аргументов. Введите `bat --help`, чтобы просмотреть список всех возможных флагов и аргументов. Также вы можете закомментировать строку с помощью `#`.
```

File: doc/README-ru.md [hunk 18]
```diff
@@ -489,19 +489,19 @@ bat --generate-config-file
 
 ## Использование `bat` в Windows
 
-`bat` в основном работает полностью из коробки, но несколько возможностей нуждаются в настройке.
+`bat` в основном работает полностью из-коробки, но несколько возможностей нуждаются в настройке.
```
Comment (gsomix) [inline doc/README-ru.md:492]:
```suggestion
`bat` полностью работоспособен "из коробки", но для некоторых возможностей могут понадобиться дополнительные настройки.
```

File: doc/README-ru.md [hunk 19]
```diff
@@ -528,15 +528,15 @@ bat() {
 
 ### Терминалы и цвета
 
-`bat` поддерживает терминалы *с* и *без* поддержки truecolor. Однако,цвета в выделении синтаксиса не оптимизированы к 8-ми битным цветам и рекоментуется использовать терминалы с поддержкой 24-ых битных цветов (`terminator`, `konsole`, `iTerm2`, ...).
+`bat` поддерживает терминалы *с* и *без* поддержки truecolor. Однако,цвета в выделении синтаксиса не оптимизированы к 8-ми битным цветам и рекомендуется использовать терминалы с поддержкой 24-ых битных цветов (`terminator`, `konsole`, `iTerm2`, ...).
```
Comment (gsomix) [inline doc/README-ru.md:531]:
```suggestion
`bat` поддерживает терминалы *с* и *без* поддержки truecolor. Однако подсветка синтаксиса не оптимизирована к 8-битным цветам, и рекомендуется использовать терминалы с поддержкой 24-битных цветов (`terminator`, `konsole`, `iTerm2`, ...).
```

File: doc/README-ru.md [hunk 20]
```diff
@@ -528,15 +528,15 @@ bat() {
...
 Смотрите [эту статью](https://gist.github.com/XVilka/8346728) для полного списка терминалов.
 
 Удостовертесь, что переменная `COLORTERM` равна`truecolor` или
-`24bit`. Иначе, `bat` не сможет определить поддержку 24 битных цветов(и будет использовать 8 битные).
+`24bit`. Иначе, `bat` не сможет определить поддержку 24-битных цветов(и будет использовать 8-битные).
```
Comment (gsomix) [inline doc/README-ru.md:535]:
```suggestion
`24bit`. Иначе `bat` не сможет определить поддержку 24-битных цветов (и будет использовать 8-битные).
```

**Comment by DuckerMan** [comment]
Yep, good point. Will add it!

**Comment by DuckerMan** [comment]
That's all, thanks for review @gsomix

**Comment by sharkdp** [comment]
Thank you @DuckerMan and @gsomix!

**Pull request merged**